docs/TELEGRAM_CUSTOM_MESSAGES.md

```diff
@@ -352,6 +352,41 @@ module.exports = botBuilder(message => {
 
 
 
+## Contact messages
+
+Contact template allows you to send a contact with phone number, first name and last name.
```

Comment:
@IvanJov could you rephrase the last words here.
`...allows you to share a contact's first name, last name and phone number`